app/data/data-claim-flow/src/main/kotlin/com/hedvig/android/data/claimflow/ClaimFlowStepExt.kt

```diff
@@ -171,6 +178,28 @@ private fun CheckoutMethodFragment.toCheckoutMethod(): CheckoutMethod {
   }
 }
 
+private fun CompensationFragment.toCompensation(): ClaimFlowDestination.Compensation {
```

Comment:
If those framents were not here, we'd then be checking here on
is FlowClaimSingleItemCheckoutRepairCompensationCompensationFragment
is FlowClaimSingleItemCheckoutValueCompensationCompensationFragment
Instead of those two fragments. Which is definitely more verbose, but we get type safety here it's not a problem of mistyping something :D